policy/scan/local_scanner.go

```diff
@@ -111,6 +132,27 @@ func NewLocalScanner(opts ...ScannerOption) *LocalScanner {
 		ls.recording = recording.Null{}
 	}
 
+	if ls.runtime == nil {
+		runtime := providers.DefaultRuntime()
+		refreshInterval := defaultRefreshInterval
+		if ls.refreshInterval > 0 {
+			refreshInterval = ls.refreshInterval
+		}
+
+		// By default, auto-update is enabled. It can be explicitly disabled
+		// by passing WithAutoUpdate(false)
+		autoUpdate := true
```

Comment:
if you assign this default value when creating the scanner struct, you will not need this logic